files/build/versions/build/build-sonic-slave-buster/versions-deb-buster

```diff
@@ -42,7 +42,7 @@ libnl-route-3-dev==3.5.0-1
 libpango1.0-dev==1.42.4-8~deb10u1
 libpangoxft-1.0-0==1.42.4-8~deb10u1
 libpixman-1-dev==0.36.0-1
-libsaibcm==4.3.0.13-1
+libsaibcm==4.3.3.1
```

Comment:
@xumia , why sonic-slave-buster contains libsaibcm?

Reply:
@lguohan , it is not the versions of sonic-slave-buster, it is for the container of the sonic-slave-buster. The package libsaibcm==4.3.3.1 is installed to the container. We put in the build folder and add a prefix "build-" at beginning of the version folder name. There is another folder for sonic slave buster, files/build/versions/**dockers**/sonic-slave-buster/versions-deb-buster.
And we have a work item https://github.com/Azure/sonic-buildimage/issues/6851 to remove the packages in the version files, which are generated in the repo itself.

Reply:
It is to support container level version control.